### PR TITLE
Add getNumberAttribute for parsing number attributes

### PR DESCRIPTION
**Pull request by mduan**

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -97,6 +97,37 @@ describe(`WebComponent instance`, function() {
...
+
+      it(`returns int number when attribute is int numeric string`, function() {
+        el.setAttribute(`num`, `123`);
+        expect(el.getNumberAttribute(`num`)).to.eql(123);
+      });
+
+      it(`returns decimal number when attribute is decimal numeric string`, function() {
+        el.setAttribute(`num`, `123.45`);
+        expect(el.getNumberAttribute(`num`)).to.eql(123.45);
+      });
```
Comment (tdumitrescu) [inline test/index.js]: 👍 <3 tests. Can we also get tests for:
- '0' -> 0 (to protect against regressions caused by JS falsiness)
- negative numbers

File: lib/index.js [hunk 2]
```diff
@@ -13,6 +13,23 @@ export default class WebComponent extends ExtendableHTMLElement {
...
+  // "asd"     -> null
+  // "123asd"  -> null
+  // "123"     -> 123
+  // "123.45"  -> 123.45
+  getNumberAttribute(attrName) {
+    const attrVal = this.getAttribute(attrName);
+    if (!attrVal) {
+      return null;
+    } else {
+      const num = Number(attrVal);
```
Comment (tdumitrescu) [inline lib/index.js]: Any particular motivations for using this method to parse, as opposed to say `parseInt`?

Comment (mduan) [reply lib/index.js]: `parseInt('123asd')` -> `123` which IMO opinion is undesired. Also, `Number` works for both int and float.

Comment (tdumitrescu) [reply lib/index.js]: fair enough 🌮 🌮

**Comment by mduan** [comment]
@tdumitrescu I'll leave it to you to publish and integrate into panel/apps at your leisure.

**Pull request merged**